scrapers/IWantClips.yml

```
@@ -0,0 +1,31 @@
+name: IWantClips
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - iwantclips.com/store
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //span[@class="headline hidden-xs"]/text()
+      Date:
+        selector: //div[@class="col-xs-12 date fix"]/span/em/text()
+        postProcess:
+          - replace:
+            - regex: Published\s(.+)
+              with: $1
+          - parseDate: Jan 2, 2006
+      Details: //div[@class="col-xs-12 description fix"]/span
+      Tags:
+        Name:
+          selector: //div[@class="col-xs-12 hashtags fix"]/span/em | //div[@class="col-xs-12 category fix"]/span
+          split: ","
```

For the split to work properly you first need to concat the results. Otherwise you'll only get the split from the first matches, in this case the `em` tags only
```suggestion
          selector: //div[@class="col-xs-12 hashtags fix"]/span/em | //div[@class="col-xs-12 category fix"]/span
          concat: ","
          postProcess:
            - replace:
                - regex: ',\s+'
                  with: ","
          split: ","
```